terraform/dev/terraform.tfvars

```
@@ -0,0 +1,29 @@
+# The amount of containers to run
+ecs_service_desired_count = "2"
+
+# The maximum of tasks that can be up during a deployment
+# use lower values for higher desired_count
+# and vice-versa.
+# Never above 200
+# Never below minimum_healthy_percent
+ecs_service_deployment_maximum_percent = "150"
+
+# The minimum of tasks that can be up during a deployment
+# we recommend you to always leave it 100 unless
+# you have a higher number (8+) of tasks running
+ecs_service_deployment_minimum_healthy_percent = "100"
+
+# --- do not edit below this line ---
+route53_record_alias_zone = "dev.creditas.io"
+
+region = "us-east-1"
+
+environment = "dev"
+
+owner = "david.montante"
+
+name = "mx-autofin-backend"
+
+squad = "auto"
```

Here our standard is PascalCase
```suggestion
squad = "Auto"
```